index.html

```diff
@@ -92,27 +92,30 @@ <h1 class="brand-heading">Breeze '16</h1>
     </header>
 
     <!-- About Ss Section -->
-    <section id="aboutUs" class="container content-section text-center">
-        <div class="row">
-            <div class="col-lg-8 col-lg-offset-2">
-                <h1>About Breeze</h1>
-                <p>
-                It’s here. It’s time to unleash your inner prowess, gather gasps, dazzle the crowds and walk away with the fame.
-                There is no talent unworthy of recognition and at Breeze ’16 you can click, write, debate, design, sing, dance, construct or program yourself to glory.<br>
-                The curtain will rise and the music will play, the spotlight will shine and the haze will fade. This is your moment to unleash your inner Cobain, Picasso, Jackson or Rowling. At Shiv Nadar University’s annual Inter-College-Sports-Techno-Cultural fest, we give you the chance to do it all.<br>
-                At Breeze ’16 we take you into virtual reality, letting you delve deep into the action-packed world of video gaming. Shining computer screens and gaming consoles are the rulers of today. This year, we show you how to bring those moments in front of your screens to life. Rescue the princess with Mario, and save the world with COD, race against speed itself with NFS. Brain and brawn combine to put you on the throne of this virtual reality.
-                </p>
-                <p><b>"CHALLENGE YOURSELF."</b><br>
-                See you at <b>Breeze ’16</b></p>
-                <h4>7<sup>th</sup> October-9<sup>th</sup> October</h4>
-                <h3><a href="http://snu.edu.in/" target="_blank">About Shiv Nadar University</a></h3>
-
-                <a href="#contact" class="btn btn-circle page-scroll">
-                    <i class="fa fa-angle-double-down animated"></i>
-                </a>                
+    <header class="bg-aboutUs">
```

Comment:
Do we really need to use another header? We can just apply the class `bg-aboutUS` to `<section id="aboutUs" class="container content-section text-center">`